include/ews/ews.hpp

```diff
@@ -10888,6 +10888,39 @@ class attachment final
         return node ? std::string(node->value(), node->value_size()) : "";
     }
 
+    //! \brief Returns the attachment's mime content
+    //!
+    //! If this is a <tt>\<ItemAttachment></tt> and the mime content was
+    //! requested, returns the Base64-encoded mime content of the attachment. If
+    //! this is an \<FileAttachment> or the mime content was not requested,
+    //! returns an empty string.
```

Comment:
And what if this is an `<FileAttachment/>` and the mime content is requested? Or, if this is an `<ItemAttachment/>` and the mime content is not requested.

Reply:
In both cases, an empty string is returned.
Maybe you misread the second sentence in the comment. In case it's FileAttachment OR the content is not requested, it will return an empty string.

Reply:
Ok! I see.